src/main/java/org/isf/patient/gui/PatientInsertExtended.java

```diff
@@ -82,31 +88,23 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * ------------------------------------------
- * PatientInsertExtended - model for the patient entry
- * -----------------------------------------
- * modification history
- * 11/08/2008 - alessandro - added mother and father names textfield
- * 11/08/2008 - alessandro - changed economicStatut -> hasInsurance
- * 19/08/2008 - mex        - changed educational level with blood type
- * 26/08/2008 - cla		   - added calendar for calculating age
- * 						   - modified age field from int to varchar
- * 28/08/2008 - cla		   - added tooltip for age field and checking name and age for patient editing
- * 05/09/2008 - alex       - added patient code
- * 01/01/2009 - Fabrizio   - modified assignment to age field to set an int value
- * ------------------------------------------
+ * ------------------------------------------ PatientInsertExtended - model for the patient entry ----------------------------------------- modification history
```

Comment:
Please undo the formatting the revision comments

Reply:
We have to agree that having devs to enable "auto-format-upon-save" IDE's feature, will make hard to avoid such cases... we should either avoid comments reformating or just remove this old way of tracking revisions, wdyt?

Reply:
I vote to remove the old way of tracking revisions; that is what `git history` is very good at doing

Reply:
Ok, I think we can open an issue for this, so we can handle also other cases, wdyt?